### DISCUSSION TITLE
Hello there

### DISCUSSION THREAD

**Discussion by ghost** [visualpython/visualpython#192] [General]
I want to contribute

**Comment by minjk-bl** [comment]
Hello @gunterno :)
We are working on supporting other environment (like colab, jupyterlab...) so it's hard to give you a part right now. 
Is there anything you want to contribute especially? We recommend these parts to contribute.
1. Bug fix: Find bug and Fix bug
2. Enhancement: Give an idea to improve Apps and Implement codes to support this idea.
3. Documentation1: Documentation & Tutorials for visualpython (with gitbook)
4. Documentation2: Writing articles(on blog) to inform Visual Python to other users.

We appreciate your contributions and want to know which part you prefer :)

**Reply by ghost** [reply]
Hello !
- Bug fix: Find bug and Fix bug
- Enhancement: Give an idea to improve Apps and Implement codes to support this idea.

**Reply by minjk-bl** [reply]
Hello, sorry for late reply.
We were working on integrating multiple-platform such as jupyterlab and colab.
Now, you can contribute as you want. Please refer to **[this document](https://bird-energy-733.notion.site/visualpython-docs-85c0274ff7564747bb8e8d77909fc8b7)** and follow **Git workflow** page to access on code level.
If there's any question and support you need, please let us know.
Thank you!
